app/grandchallenge/ai_website/admin.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
+
+from grandchallenge.ai_website.models import (
+    CompanyEntry,
+    ProductEntry,
```

Comment:
I would remove the `Entry` suffix from all of the models, again, naming sticks around in django.